corehq/apps/hqwebapp/static/hqwebapp/js/hqModules.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * hqModules provides a poor man's module system for js. It is not a module *loader*,
+ * only a module *referencer*: "importing" a module doesn't automatically load it as
+ * a script to the page; it must already have been loaded with an explict script tag.
+ *
+ * Modules MUST have a name, and SHOULD be given then name
```

Comment:
very minor:

s/then name/the name/

also, inconsistent wrapping with previous paragraph